packages/user-interface/src/pages/TaskPage.tsx

```diff
@@ -12,21 +12,25 @@ import {
 import "font-awesome/css/font-awesome.min.css";
 import { Alert } from "@gemeente-denhaag/components-react";
 import { useIntl } from "react-intl";
-import useQuery from "../hooks/useQuery";
 import "./TaskPage.module.scss";
+import { useParams } from "react-router-dom";
+import BackLink, { BackLinkProps } from "../components/BackLink";
 
-const TaskPage = () => {
-  const query = useQuery();
+interface TaskPageProps {
+  backlink?: BackLinkProps;
```

Comment:
Is er in dit geval niet altijd een backLink nodig? Of hebben je nog nieuwe inzichten?

Reply:
Wat mij betreft niet. In de huidige TaskPage zit ook geen backlink, dus lijkt mij prima op deze optioneel te houden.